stages/org.osbuild.systemd-sysusers.d.meta.json

```diff
@@ -0,0 +1,103 @@
+{
+  "summary": "Create system users via systemd-sysusers.",
+  "description": [
+    "This stage allows to copy systemd-sysusers configuration files and create them.",
+    "It accepts a list of paths which are copied to the sysuser.d directory as well",
+    "as an inline configuration of a new sysusers configuration file. The `filename`",
+    "property .conf file to be added. The fields for each sysuser entry are validated",
+    "as specified by sysusers.d(5)."
```

Comment:
I found this dual behaviour a bit confusing when I was looking at the test data.  I don't see the reason to support copying files in this stage.  That can be handled very easily with a regular old copy stage.  The fact that this stage can do both writing of a config and copying of files in a single invocation makes it even harder to follow, in my opinion.

Writing these config files with stage options is of course a great feature and a long time coming.  Thank you for this.  But I would prefer if the `copy` feature was dropped.

Reply:
Initially, I thought of it as a - kind of - convenience feature to also support paths as well. However, to have any additional value than just copying, it should've validated the .conf files are valid sysuser files - which I didn't add (and I am not sure it makes much sense).
So I agree and removed the `paths` option from the systemd-sysusers.d stage. 
@achilleas-k PTAL